site/docs/reference/Connectors/capture-connectors/alpaca.md

```diff
@@ -0,0 +1,116 @@
+| **`/start_date`** | Start Date | Get trades starting at this date. Has no effect if changed after the capture has started. | string | Required |
+| **`/symbols`** | Symbols | Comma separated list of symbols to monitor. | string | Required |
+
+#### Bindings
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **`/name`** | Name | Unique name for this binding. Cannot be changed once set. | string | Required |
+
+### Sample
+
+```yaml
+captures:
+  ${PREFIX}/${CAPTURE_NAME}:
+      endpoint:
+        connector:
+          image: "ghcr.io/estuary/source-alpaca:dev"
+          config:
+            api_key_id: <SECRET>
+            api_secret_key: <SECRET>
+            feed: iex
+            start_date: 2022-11-01T00:00:00Z
+            symbols: AAPL,MSFT,AMZN,TSLA,GOOGL,GOOG,NVDA,BRK.B,META,UNH
+            advanced:
+              is_free_plan: true
+      bindings:
+        - resource:
+            name: trades
+          target: ${PREFIX}/${CAPTURE_NAME}/trades
+```
+
+## Limitations
+
+#### Capturing data for more than 20 symbols in a single capture could result in API errors.
+
+If you need to capture data for more than 20 symbols, we recommend splitting them between two captures.
+Support for a larger number of symbols in a single capture is planned for a future release.
+
+#### Separate historical and real-time data streams will result in some duplicate trade documents.
+
+As discussed [above](#real-time-and-historical-trade-data), the connector captures historical and real-time data in two different streams.
+As the historical data stream catches up to the present, it will overlap with the beginning of the real-time data stream, resulting in some duplicated documents.
+These will have [identical properties from Alpaca](https://alpaca.markets/docs/api-references/market-data-api/stock-pricing-data/historical/#response-object-properties), but different [metadata from Flow](../../../concepts/collections.md#documents).
+
+There are several ways to resolve this:
+
+* If you plan to materialize to an endpoint for which standard (non-delta) updates are supported, Flow will resolve the duplicates during the materialization process.
+Unless otherwise specified in their [documentation page](../materialization-connectors/README.md), materialization connectors run in standard updates mode.
+If a connector supports both modes, it will default to standard updates.
+
+* If you plan to materialize  to an endpoint for which [delta updates](../../../concepts/materialization.md#delta-updates) is the only option,
+ensure that the endpoint system supports the equivalent of [lastWriteWins](../../reduction-strategies/firstwritewins-and-lastwritewins.md) reductions.
+Alternatively, use a [derivation](../../../guides/create-derivation.md) to resolve the duplicates before materializing.
```

Comment:
We don't currently have a practical way to resolve duplicates with a derivation, so we could nix this last line.